{
    "affected": [
        {
            "package": {
                "ecosystem": "photon:4.0",
                "name": "python3-urllib3",
                "purl": "pkg:rpm/vmware/python3-urllib3?distro=photon-4"
            }
        },
        {
            "package": {
                "ecosystem": "photon:4.0",
                "name": "rubygem-bundler",
                "purl": "pkg:rpm/vmware/rubygem-bundler?distro=photon-4"
            }
        },
        {
            "package": {
                "ecosystem": "photon:4.0",
                "name": "zeromq",
                "purl": "pkg:rpm/vmware/zeromq?distro=photon-4"
            }
        }
    ],
    "id": "PHSA-2021-4.0-0060",
    "modified": "2023-03-16T05:25:30Z",
    "published": "2021-03-12T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://github.com/vmware/photon/wiki/Security-Update-4.0-4"
        }
    ],
    "related": [
        "CVE-2019-3881",
        "CVE-2020-36400",
        "CVE-2021-33503",
        "CVE-2020-36327"
    ]
}